tests/QssTestDefs.js

```diff
@@ -1654,8 +1654,8 @@ gpii.tests.qss.testDefs = {
         undoTestSequence,
         qssCrossTestSequence,
         stepperindicatorsSequence,
+        restartWarningSequence,
         crossQssTranslations,
-        appZoomTestSequence,
```

Comment:
Any reason these were reordered? Perhaps they are in reverse alphabetical order now? : P

Reply:
In reality we don't know the reason, but only this order the tests run properly. Maybe some of them are entangled to each other. We are talking about fully revisiting all of the tests of some point, because some (most) of them are legacy at this point. But this will be another time. As for now they need to be ordered this way for no apparent reason so we need to bear with it 😈 .

Reply:
OK - please add a comment to this effect as well : P